src/Infrastructure/Controller/MyArea/Profile/DetailDeleteProfileController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\MyArea\Profile;
+
+use App\Infrastructure\Security\AuthenticatedUser;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+final class DetailDeleteProfileController
+{
+    public function __construct(
+        private AuthenticatedUser $authenticatedUser,
+        private \Twig\Environment $twig,
+    ) {
+    }
+
+    #[Route('/profile/delete/detail', name: 'app_profile_delete_detail', methods: ['GET'])]
+    public function __invoke(Request $request): Response
+    {
+        $user = $this->authenticatedUser->getUser();
+
+        return new Response(
+            content: $this->twig->render(
+                name: '/my_area/profile/delete-profile.html.twig',
+                context: [
+                    'user' => $user,
```

Comment:
Ici tu n'as pas forcément besoin de passer l'utilisateur connecté au template twig. Tu peux l'avoir directement en utilisant `{{ app.user }}`

Reply:
j'ai cette erreur quand je retire le context avec le user et que j'utilise app.user dans twig : "Object of class App\Infrastructure\Security\SymfonyUser could not be converted to string"

Reply:
Oui parce que c'est un objet `SymfonyUser`. Derrière tu peux du coup appeler les méthodes de cette classes. Par exemple, `{{ app.user.getFullName() }}`